### ISSUE TITLE
Databases with Views produce errors and fail to backup

### ISSUE THREAD

**Issue by BrandonRoehl** [jamf/go-mysqldump#8]
**Steps to recreate**
1. Create the view
2. Backup the database

Example view
```sql
CREATE VIEW testview AS SELECT 'test' as c1, 'other as c2;
```

Error with line number. This happends in the `CreateSQL` function in `dump.go`
```
template:mysqldumpTable:9.3: executing "mysqldumpTable" at <.CreateSQL>:error calling CreateSQL: sql: expected 4 destination arguments in Scan, not 2
```

**Comment by billyaustin84** [comment]
Hey @BrandonRoehl have you ever looked into what would be required to resolve this?

**Comment by BrandonRoehl** [comment]
Yes the first of 2 PRs is now open you can see it here at #11 after the virtual column fix. I am going to need to make a new PR to do the same dynamic scanning for the show sql statement since the columns are not predetermined
